docs/publishing/legacy-publishing-nanopublications.md

```diff
@@ -1,30 +1,30 @@
 # Publishing nanopublications
-The `nanopub` library provides an intuitive API that makes publishing nanopublications much easier. 
-The rationale is that you often do not want to worry about the details of composing 
-the RDF that is often the same in each nanopublication. Instead you should focus on the 
+The `nanopub` library provides an intuitive API that makes publishing nanopublications much easier.
+The rationale is that you often do not want to worry about the details of composing
+the RDF that is often the same in each nanopublication. Instead you should focus on the
 content of your nanopublication: the assertion.
 
 ## Prerequisits for publishing
-Before you can publish you should [setup your profile](../getting-started/setup)
+Before you can publish you should [setup your profile](/nanopub/getting-started/setup)
 
 ## Quickly publishing nanopublications using `claim`
 You can quickly publish a nanopublicaiton with a single simple statement using the `claim` method:
 ```python
->>> from nanopub import NanopubClient
+from nanopub import NanopubClient
 
->>> # Create the client (we use use_test_server=True to point to the test server)
->>> client = NanopubClient(use_test_server=True)
+# Create the client (we use use_test_server=True to point to the test server)
+client = NanopubClient(use_test_server=True)
```

Comment:
This is the legacy doc, so I did not updated it and it might not be right :p 

If the `NanopubConf` is not set at all then a default is generated, but the profile is set to None by default

I set it this way to avoid having errors on default settings that depends on the environment

You will need to do this to load the default profile:
```python
conf = NanopubConf(profile=load_profile())
```

Reply:
Ah I see, yes, thanks